mk/cfg/x86_64-unknown-linux-uclibc.mk

```diff
@@ -0,0 +1,29 @@
+# x86_64-unknown-linux-uclibc configuration
```

Comment:
If we exclusively use rustbuild (which is how I've tested this before), then this file is not needed at all.